### PR TITLE
 #159349450 Build out  controller for `create entry route`

### PR DESCRIPTION
**Pull request by blackshady**
#### What does this PR do?
Create controller to handle create entry route
#### Description of Task to be completed?
- Secure route
- catch some error thrown by route using a error hander
- create a createEntry method in controller
- Validate `createEntry route`  `login and signup` 
- remove un-used data
- resolve LFA Feedback
    * remove houndci
    * correct status code mistake
    * remove moment libary
    * rename .json file to js file

#### How should this be manually tested?
None
#### Any background context you want to provide?
- This route is yet to have a validation to capture other edge case for error

#### What are the relevant pivotal tracker stories?
[#159349450](https://www.pivotaltracker.com/story/show/159349450/comments/192489671)
#### Screenshots (if appropriate)
None
#### Questions:
None

### CODE REVIEW

File: Server/src/controllers/EntriesController.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import moment from 'moment';
 import database from '../config/databaseConnection';
 import find from '../models/queries/find.json';
 import entriesDb from '../models/dummy-db/Entries.json';
```
Comment (fob413) [inline Server/src/controllers/EntriesController.js]: Why do you have files in  `json`?

File: Server/src/controllers/EntriesController.js [hunk 2]
```diff
@@ -26,15 +27,15 @@ class EntriesController {
       rows,
     } = await database.query(find.userEntries, [userid]);
     if (rows.length === 0) {
-      res.status(404).json({
+      return res.status(404).json({
```
Comment (fob413) [inline Server/src/controllers/EntriesController.js]: a users should not get a `404` error in a getAll request

File: Server/src/controllers/EntriesController.js [hunk 3]
```diff
@@ -68,38 +69,35 @@ class EntriesController {
...
       story,
     } = req.body;
-    let userId, entryId;
-    if (entriesDb.length === 0) {
-      userId = 1;
-      entryId = 1;
-    } else {
-      userId = entriesDb.length + 1;
-      entryId = entriesDb.length + 1;
+    if (!title && !story) {
```
Comment (fob413) [inline Server/src/controllers/EntriesController.js]: Shouldn't this be in a validator?

File: Server/src/controllers/EntriesController.js (hunk 2 above)
Comment (fob413) [inline Server/src/controllers/EntriesController.js]: a users should not get a `404` error in a getAll request

File: Server/src/controllers/EntriesController.js [hunk 4]
```diff
@@ -68,38 +69,35 @@ class EntriesController {
...
+      return res.status(400).json({
+        status: 'error',
+        message: 'Bad Request, fail to save entry, filed cannot be empty',
+      });
     }
     const createdAt = moment().format('MMMM DD YYYY, h:mm:s A z').trim();
```
Comment (fob413) [inline Server/src/controllers/EntriesController.js]: You are not to use `moment` library

File: Server/src/controllers/EntriesController.js (hunk 1 above)
Comment (blackshady) [reply Server/src/controllers/EntriesController.js]: entriesDb is a file from my dummy database I will flush out ASAP,

**Review by fob413 — APPROVED** [review]

**Pull request merged**